src/plugins/core/spreadsheet_pivot.ts

```diff
@@ -39,4 +56,14 @@ export class SpreadsheetPivotCorePlugin extends CorePlugin {
       }
     }
   }
+
+  private checkDataSetValidity(definition: PivotCoreDefinition) {
+    if (definition.type === "SPREADSHEET") {
+      const { zone, sheetId } = definition.dataSet || {};
+      if ((sheetId && !this.getters.tryGetSheet(sheetId)) || (zone && !isZoneValid(zone))) {
```

Comment:
isn't the pivot also invalid is zone/sheetId/definition.dataSet is undefined ? We will return `CommandResult.Success` in those cases if I read the condition correctly